tests/generation/test_utils.py

```diff
@@ -1989,32 +1994,6 @@ def test_generate_continue_from_inputs_embeds(self):
                         )
                     )
 
-    @parameterized.expand([("offloaded",)])  # ("offloaded_static",) TODO: @raushan fixme in some models (eg T5)
-    @require_torch_accelerator
-    @pytest.mark.generate
-    def test_offloaded_cache_implementation(self, cache_implementation):
```

Comment:
this test doesn't make sense? tests that two `generate` calls with the same flags have the same output

We have tests for the offloaded cache in `tests/utils/test_cache_utils.py`

Reply:
There are slow tests for all caches on `main`, [this PR](https://github.com/huggingface/transformers/pull/37394) adds a fast test for all caches 🤗